code/mission/missionbriefcommon.cpp

```diff
@@ -1118,7 +1118,6 @@ void brief_render_icon(int stage_num, int icon_num, float frametime, int selecte
 		if ( ( (bx < 0) || (bx > gr_screen.max_w_unscaled) || (by < 0) || (by > gr_screen.max_h_unscaled) ) && !Fred_running ) {
 			bi->x = bx;
 			bi->y = by;
-			return;
 		}
```

Comment:
Since bi->x/y is assigned at the end of the function, this entire if block can just be removed if the return is removed

Reply:
Good point. Done.